### ISSUE TITLE
The question about BBCU-M

### ISSUE THREAD

**Issue by TIEHua** [Zj-BinXia/BBCU#1]
Hi, thank you for sharing the code.  I have some questions about when running the code.
1. I noticed that in the configuration file of your code, gt_size is set to 128. In the paper, it is size of 192×192 randomly cropped from training data. Which should we use?
2. I tested the BBCU-M when it was set to 192x192. Finally, the PNSR obtained on Set5 and Set14 is 31.4863 and 28.1566 (x4), which are lower than the results in the original paper. Could you please help me to check my configuration file？
Thanks for your time. I am looking forward to your reply.

name: Train_BBCUM_x4
model_type: SRModel
scale: 4
num_gpu: 1  # set num_gpu: 0 for cpu mode
manual_seed: 0

datasets:
  train:
    name: DIV2K
    type: PairedImageDataset
    dataroot_gt: /scratch/project/DIV2K/DIV2K_train_HR_sub
    dataroot_lq: /scratch/project/DIV2K/DIV2K_train_LR_bicubic/X4_sub
    # (for lmdb)
    # dataroot_gt: datasets/DIV2K/DIV2K_train_HR_sub.lmdb
    # dataroot_lq: datasets/DIV2K/DIV2K_train_LR_bicubic_X4_sub.lmdb
    filename_tmpl: '{}'
    io_backend:
      type: disk
      # (for lmdb)
      # type: lmdb

    gt_size: 192
    use_hflip: true
    use_rot: true

   
    num_worker_per_gpu: 6
    batch_size_per_gpu: 16
    dataset_enlarge_ratio: 100
    prefetch_mode: ~

  val:
    name: Set5
    type: PairedImageDataset
    dataroot_gt: /scratch/project/benchmark/Set5/HR
    dataroot_lq: /scratch/project/benchmark/Set5/LR_bicubic/X4
    filename_tmpl: '{}x4'
    io_backend:
      type: disk

  val_2:
    name: Set14
    type: PairedImageDataset
    dataroot_gt: /scratch/project/benchmark/Set14/HR
    dataroot_lq: /scratch/project/benchmark/Set14/LR_bicubic/X4
    filename_tmpl: '{}x4'
    io_backend:
      type: disk

network_g:
  type: BBCUM
  num_in_ch: 3
  num_out_ch: 3
  num_feat: 64
  num_block: 16
  upscale: 4
  img_range: 130.

path:
  pretrain_network_g: ~
  param_key_g: params
  strict_load_g: true
  resume_state: ~

train:
  ema_decay: 0
  optim_g:
    type: Adam
    lr: !!float 1e-4
    weight_decay: 0
    betas: [0.9, 0.99]
  
  scheduler:
    type: MultiStepLR
    milestones: [200000]
    gamma: 0.5

  total_iter: 300000
  warmup_iter: -1  # no warm up

  
  pixel_opt:
    type: L1Loss
    loss_weight: 1.0
    reduction: mean

val:
  val_freq: !!float 5e3
  save_img: false

  metrics:
    psnr: # metric name, can be arbitrary
      type: calculate_psnr
      crop_border: 4
      test_y_channel: true
      better: higher  # the higher, the better. Default: higher

logger:
  print_freq: 100
  save_checkpoint_freq: !!float 5e3
  use_tb_logger: true
  wandb:
    project: ~
    resume_id: ~

dist_params:
  backend: nccl
  port: 29500

**Comment by Zj-BinXia** [comment]
1.应该设定在128
2.我测试出来就是,BBCUM x4没有问题
Set5:
       # psnr: 31.5404        
       # ssim: 0.8862 

Set14：
         # psnr: 28.2001       
         # ssim: 0.7718

**Comment by TIEHua** [comment]
> 1.应该设定在128 2.我测试出来就是,BBCUM x4没有问题 Set5: # psnr: 31.5404 # ssim: 0.8862
> 
> Set14： # psnr: 28.2001 # ssim: 0.7718

谢谢您的回复，我再测试下
